mycroft/skills/npr_news/test/intent/sample2.intent.json

```diff
@@ -1,7 +1,7 @@
 {
-  "utterance": "play the news",
+  "utterance": "is there any news",
```

Comment:
No, there's no conflict between the actual vocab in the NPR news skill and these play skills. The only conflict was in a hypothetical `play the news